cms/settings/base.py

```diff
@@ -169,6 +171,9 @@
                 "wagtail.contrib.settings.jinja2tags.settings",
                 "cms.core.jinja2tags.CoreExtension",
             ],
+            "policies": {
```

Comment:
suggestion: link to the Jinja docs for context
```suggestion
            "policies": {
                # https://jinja.palletsprojects.com/en/stable/api/#policies
```